app/routes/chatbot.js

```diff
@@ -260,7 +267,7 @@ router.use((req, res, next) => {
     dashbotLog = `${dashbotLog}${req.incoming_message}`;
   }
 
-  req.user.postDashbotIncoming(dashbotLog);
+  req.user.postDashbotIncoming(dashbotLog.toLowerCase());
```

Comment:
Storing incoming messages as lowercase to avoid logging Yes/yes, No/no, START/start/Start, etc, as separate stats.